fdbserver/BlobManager.actor.cpp

```diff
@@ -5804,15 +5804,16 @@ ACTOR Future<Void> blobManager(BlobManagerInterface bmInterf,
 		self->addActor.send(monitorClientRanges(self));
 		self->addActor.send(monitorTenants(self));
 		self->addActor.send(monitorPurgeKeys(self));
-		if (SERVER_KNOBS->BG_CONSISTENCY_CHECK_ENABLED) {
-			self->addActor.send(bgConsistencyCheck(self));
-		}
+
 		if (SERVER_KNOBS->BG_ENABLE_MERGING) {
 			self->addActor.send(granuleMergeChecker(self));
 		}
 		self->addActor.send(backupManifest(self));
 		self->addActor.send(truncateMutationLogs(self));
 
+		if (!BUGGIFY && !self->isFullRestoreMode) {
```

Comment:
If its not a restore why do we only run the consistency check if buggify is off? Do we want to maintain similar behaviour to before and do: `if (randomize && BUGGIFY && !self->isFullRestoreMode)`?

Reply:
I Added back the knob for possible later use, so now when `randomize && BUGGIFY` we have BGCC off as before. The same behaviour maintains. Thanks.